.github/amlarc/amlarc-tool.sh

```diff
@@ -2,8 +2,9 @@
 set -x
 
 # Global variables
-export LOCK_FILE=$0.lock
-export RESULT_FILE=amlarc-test-result.txt
+export SCRIPT_DIR=$( cd  "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+export LOCK_FILE=${SCRIPT_DIR}/$0.lock
+export RESULT_FILE=${SCRIPT_DIR}/amlarc-test-result.txt
 export MAX_RETRIES=60
 export SLEEP_SECONDS=20
 
```

Comment:
as we're moving it, I would suggest that we rename it to kubernetes-tool.sh, at least the folder name? @Bozhong68 @Zhong-J what do your think?

Reply:
Agree. "amlarc" wording should be replaced by "kubernetes". After GA, we will not expose amlarc to customers.